common/src/main/java/gov/cms/ab2d/common/model/Attestation.java

```diff
@@ -32,5 +32,8 @@
     @Column(columnDefinition = "TIMESTAMP WITH TIME ZONE")
     private OffsetDateTime attestedOn;
 
-
+    @Override
+    public int compareTo(Attestation other) {
+        return this.getAttestedOn().compareTo(other.getAttestedOn());
+    }
```

Comment:
In general an implementation of `compareTo` should be consistent with `equals`, which is not quite the case here. This `compareTo` will equate two completely different attestation records as long as they happen to have the same attestation date. (more on this [here](https://docs.oracle.com/javase/8/docs/api/java/lang/Comparable.html)).

I'd recommend dumping this method and using an explicit `Comparator` wherever you need it.

Reply:
Probably won't need comparisons anymore, since there will only be one contract for attestation, I can just drop this.